packages/components/tests/dummy/app/templates/components/avatar.hbs

```diff
@@ -0,0 +1,76 @@
+{{page-title "Avatar Component"}}
+
+<h2 class="dummy-h2">Avatar</h2>
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Renders an avatar; a user image if provided, otherwise a user icon by default.</p>
+
+</section>
```

Comment:
super nit: Since content is indented, this blank line before the closing parent tag could be removed.